### PR TITLE
[12.0][MIG] account_analytic_distribution_required

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT17619

### CODE REVIEW

File: account_analytic_distribution_required/models/account.py [hunk 1]
```diff
@@ -24,15 +24,15 @@ class AccountMoveLine(models.Model):
     @api.multi
     def _check_analytic_distribution_required_msg(self):
         for move_line in self:
-            if move_line.analytic_account_id \
-                    and move_line.analytic_distribution_id:
+            analytic_distribution = move_line.analytic_tag_ids.filtered(
+                "active_analytic_distribution")
+            if move_line.analytic_account_id and analytic_distribution:
```
Comment (pedrobaeza) [inline account_analytic_distribution_required/models/account.py:29]: Is this constraint still valid? I don't think so.

File: account_analytic_distribution_required/models/account.py [hunk 2]
```diff
@@ -64,7 +63,10 @@ def _check_analytic_distribution_required_msg(self):
                          move_line.analytic_account_id.code,
                          move_line.analytic_account_id.name)
 
-    @api.constrains('analytic_account_id', 'analytic_distribution_id',
+    def write(self, vals):
+        super().write(vals)
```
Comment (pedrobaeza) [inline account_analytic_distribution_required/models/account.py]: Why this?

File: account_analytic_distribution_required/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+#. Create an invoice and add a line with an account of the same type you
+   are configured above.
+#. Add an analytic distribution or an analytic account to that line.
```
Comment (pedrobaeza) [inline account_analytic_distribution_required/readme/USAGE.rst]: This text must change according new way of distributions through tags.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_analytic_distribution_required/models/account.py (hunk 1 above)
Comment (pedrobaeza) [reply account_analytic_distribution_required/models/account.py:29]: Seen by chat. We keep it.

File: account_analytic_distribution_required/models/account.py (hunk 2 above)
Comment (ernestotejeda) [reply account_analytic_distribution_required/models/account.py]: It was a mistake. Removed

**Comment by ernestotejeda** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**